lib/airbrake/tasks/airbrake.cap

```diff
@@ -18,6 +18,8 @@ namespace :airbrake do
           notify_command << " REVISION=#{fetch(:current_revision)} REPO=#{fetch(:repo_url)}"
           notify_command << " USER=#{local_user.strip.shellescape}"
           notify_command << " API_KEY=#{ENV['API_KEY']}" if ENV['API_KEY']
+          notify_command << " DEPLOY_MESSAGE=#{ENV['DEPLOY_MESSAGE']}" if ENV['DEPLOY_MESSAGE']
+          notify_command << " RAILS_ENV=#{fetch(:rails_env)}"
```

Comment:
we already set `rails_env` using [`ENV['TO']`](https://github.com/astikans/airbrake/blob/deploy_message/lib/airbrake/tasks/airbrake.cap#L17).